app/assets/javascripts/app/settings/profile.js

```diff
@@ -30,7 +30,11 @@ angular.module('app').controller('Settings', function($scope, $routeParams, $api
       if (updated_person.error) {
         $scope.error = updated_person.error;
       } else {
-        $scope.success = 'Settings have been saved.';
+        if (updated_person.email === $scope.form_data.email) {
+          $scope.success = 'Settings have been saved.';
+        } else {
+          $scope.success = "An email has sent to your account to verify email changed.";
```

Comment:
You will receive an email with instructions for how to confirm your email address in a few minutes.